mantidproject/mantid
# Bug in FindDetectorsOutsideLimits

Description of work.
This ticket is part of changes necessary for #16808 to work

Fixes #16998

**To test:**

Run the script provided for  the ticket and look for correct behaviour. The bug, changes and solution are trivial, so testing should be easy. 

Nobody noticed this bug until I've fiddled with #16808, so no need in release notes.

As masking have slightly changed, I had to change references files in system tests using these reference files.  I have verified that only masking in ref-files have changed.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
